Gemfile

```diff
@@ -30,7 +30,7 @@ gem "gettext_i18n_rails_js",          "~>1.1.0"
 gem "google-api-client",              "~>0.8.6",       :require => false
 gem "hamlit",                         "~>2.0.0",       :require => false
 gem "hamlit-rails",                   "~>0.1.0"
-gem "hashie",                         "=3.4.4",        :require => false
+gem "hashie",                         "~>3.4.6",       :require => false
```

Comment:
Should this be `>= 3.4.6"` since it's not a direct dependency.  We only want to ensure that it has to be at least 3.4.6?  We should still let omniauth and other gems dictate the requirement.